trunk/SUAVE/Plots/Airfoil_Plots.py

```diff
@@ -6,56 +6,137 @@
 # ----------------------------------------------------------------------
 #  Imports
 # ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units
+import numpy as np 
 import matplotlib.pyplot as plt  
+import matplotlib.cm as cm
 
 ## @ingroup Plots
 def plot_airfoil_properties(ap,line_style= 'k-',arrow_color = 'r',plot_pressure_vectors = False ):  
-  
-    fig  = plt.figure('Airfoil')
+    
+    # determine dimension of angle of attack and reynolds number 
+    nAoA = len(ap.AoA)
+    nRe  = len(ap.Re)
+    
+    # create array of colors for difference reynolds numbers 
+    colors  = cm.rainbow(np.linspace(0, 1,nAoA))
+    markers = ['o','v','s','P','p','^','D','X','*']
```

Comment:
you can easily run out of markers (although the plot would look like uninterpretable). Use a repeat or something to add more if necessary.

Reply:
I used the modulus operator to loop repeat loop